api/tests/unit/scripts/helpers/files/organizations-pro-with-tags-and-target-profiles-test.csv

```diff
@@ -1,2 +1,2 @@
-externalId;name;provinceCode;credit;email;locale;tags;createdBy;documentationUrl;targetProfiles
-TUTU001;Team Accès;123;;   team-acces@e  x  ample.net   ;;PUBLIC_AGRICULTURE;100;https://pix.fr/;123456
+type,externalId,name,provinceCode,credit,emailInvitations,emailForSCOActivation,organizationInvitationRole,locale,tags,createdBy,documentationUrl,targetProfiles,isManagingStudents,identityProviderForCampaigns
+pro,TUTU001,Team Accès,123,,   team-acces@e  x  ample.net   ,superadmin@example.net,admin,,PUBLIC_AGRICULTURE,100,https://pix.fr/,123456,,pole_emploi
```

Comment:
1. Ne serait-ce pas pertinent de mettre dans ce fichier CSV d'autres types d'organisation qu'uniquement des organisation `pro` ?
2. Ne serait-ce pas pertinent de renommer ce fichier `organizations-pro-with-tags-and-target-profiles-test.csv` en `organizations-with-tags-and-target-profiles-test.csv` pour signifier que ce fichier n'est plus spécifique à un type d'organisation unique ?

Mais peut-être ai-je mal compris ?

Reply:
en effet j'enleverai le pro, mais pour le reste, c'est pas necessaire parce que c'est purement utiliser pour un test unitaire